src/main/java/uk/gov/hmcts/reform/managecase/api/payload/CaseAssignmentRequest.java

```diff
@@ -21,8 +23,9 @@
     private String caseTypeId;
 
     @JsonProperty("case_id")
-    @NotEmpty(message = "Case ID can not be empty")
-    @Pattern(regexp = "\\d+", message = "Case ID should contain digits only")
+    @NotEmpty(message = ValidationError.CASE_ID_EMPTY)
```

Comment:
Luhn check already assures not empty?

Reply:
@MSancaktutar  You are correct that Luhn check does assure non-empty: however `@NotEmpty` provides additional validation for when the value is null or the property has not been supplied.

**Therefore can you confirm if the current proposed solution of using all three annotations is acceptable so this conversation can be resolved?**: 

```
    @NotEmpty(message = ValidationError.CASE_ID_EMPTY)
    @Size(min = 16, max = 16, message = ValidationError.CASE_ID_INVALID_LENGTH)
    @LuhnCheck(message = ValidationError.CASE_ID_INVALID, ignoreNonDigitCharacters = false)
```

Reply:
Ok.